config/routes.rb

```diff
@@ -21,9 +21,10 @@
     resources :lines, only: [:new, :create]
   end
   root :to => redirect('/users/sign_in')
-  devise_for :users, skip: [:registrations]
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
+  # devise_for :users, skip: [:registrations]
```

Comment:
Are these routing changes essential or can we go back to skipping most of the registration stuff? I think we might be able to get away with `devise_for :users, controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }, skip: [:registrations]`